_pages/docs/request-match-rules/templated-responses.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: single
+title: Templated Responses
+permalink: /docs/templated-responses
+sidebar:
+  nav: 'docs'
+classes: wide
+---
+
+Templated responses allow responses to be dynamically created by using Scriban's liquid template API. The responses can be derived from the response's body, and/or a random value using the [faker](https://github.com/bchavez/Bogus) API.
+
+### How do I add a templated response?
+
+1. In the designer, create a new scenario or select an existing one.
+2. Open the response tab.
+3. Set the type of the response to "Templated".
+4. As an example, set the body of the response to "\{\{rant.reviews\}\}".
+5. Save the scenario, and upload it to the server.
+6. Make request to the endpoint that contains the templated response.
+
+![Templated Response](../../../assets/images/request-match-rules/add-templated-response.png)
+
+Figure 1. Adding a new templated rule. Note the checkmark next to "Templated".
+
+The server will respond with a rant review string generated from the faker API. When composing a templated scenario body, the body can contain multiple templated responses, such as "\{\{rant.reviews\}\} \{\{person.first\_name\}\}", and can contain text between and around the templates: "\{\{rant.reviews\}\} abc \{\{person.first\_name\}\}". These templates can also be nested inside of JSON. For example, \{"key":"\{\{person.first\_name\}\}"\}.
```

Comment:
this is a general comment. can we remove faker/bogus and scriban references in the article

Reply:
Fixed